packages/sign_in_with_apple/README.md

```diff
@@ -1,9 +1,10 @@
 # Sign in With Apple
 
-Flutter bridge to Sign in with Apple.
-
+Flutter bridge to Sign in with Apple.\
```

Comment:
Never seen this. The "Markdown way" would be 2 whitespaces at the end for do a line break.

Reply:
```suggestion
Flutter bridge to Sign in with Apple.  
```

But this is more obvious, while not reading as nicely / text-y.

Reply:
I never knew the two whitespaces, I always used \